src/modules/mavlink/mavlink_main.h

```diff
@@ -589,15 +589,16 @@ class Mavlink : public ModuleParams
 	 * logic will send parameters from the current index
 	 * to len - 1, the end of the param list.
 	 */
-	unsigned int		_mavlink_param_queue_index;
+	unsigned int		_mavlink_param_queue_index{0};
+
+	bool			mavlink_link_termination_allowed{false};
```

Comment:
that one should also start with an underscore, right?

Reply:
I'm glad you wanted that changed.  Done!  I also deleted 3 unnecessary struct specifiers in the same commit. Thank you for your review!